### PR TITLE
feat(routes): add normalized strings to context

### PR DESCRIPTION
**Pull request by dnlup**
This should help because it already exposes the sanitised strings if you want to use your own `getLabel` function.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -182,18 +183,29 @@ module.exports = fp(
...
             if (mode === 'dynamic') {
                 let getLabel =
                     metricsConfig.routes.getLabel || dynamicMode.getLabel;
                 if (typeof getLabel !== 'function') {
                     throw new Error('"getLabel" must be a function.');
                 }
-                fastify.decorateRequest('metricsLabel', '');
-                fastify.decorateReply('metricsLabel', '');
+                fastify.decorateRequest('_metricsLabel', '');
+                fastify.decorateReply('_metricsLabel', '');
```
Comment (dnlup) [inline index.js:203]: I would like to try a Symbol here, I am no sure what will happen with the encapsulation.

Comment (dnlup) [reply index.js:203]: After some tests, it looks like it doesn't break anything. I'll try it.

Comment (dnlup) [reply index.js]: Done. The reason behind this is that I want to expose a unified way to access the label (#84 ) and hide the internal field used by the plugin.

**Review by antoniomuso — APPROVED** [review]

**Pull request merged**